flow/workflows/cdc_flow.go

```diff
@@ -216,7 +216,8 @@ func CDCFlowWorkflowWithConfig(
 			RetryPolicy: &temporal.RetryPolicy{
 				MaximumAttempts: 20,
 			},
-			SearchAttributes: mirrorNameSearch,
+			SearchAttributes:    mirrorNameSearch,
+			WaitForCancellation: true,
```

Comment:
I set `WaitForCancellation` on everything to try get `CancelWorkflow` to pause longer, but pretty sure it didn't help, need to test with removing this now that rest is working. Not sure if we want this set to true or not now

Reply:
Tested it out with cancelling a few sync flows and it's fine to keep it (or remove it - whichever helps more).